tests/flows/test_flow.py

```diff
@@ -17,6 +17,7 @@ def check_flow(flow):
         for flow in flows:
             check_flow(flow)
 
+    @unittest.skip('Not tested until test sentinels are added back.')
```

Comment:
Flow names should be unique, [this commit](https://github.com/openml/openml-python/pull/150/commits/7e352c1f4079e86e76ed9f1217cf22277bf879bd) adds sentinels to make sure during each test creates a flow with a unique name.